e2e-tests/security-context/compare/statefulset_sec-context-proxysql-oc.yml

```diff
@@ -0,0 +1,170 @@
+apiVersion: apps/v1
```

Comment:
this file is identical to `e2e-tests/security-context/compare/statefulset_sec-context-proxysql.yml` so I think it is not needed